SayTheirNames/Source/Controller/Home/HomeController.swift

```diff
@@ -71,13 +71,19 @@ final class HomeController: UIViewController {
     private var peopleCollectionView: UICollectionView { homeView.peopleCollectionView }    
     
     // MARK: - ClASS METHODS
-    override func loadView() {
-        self.view = homeView
-        homeView.peopleDataSource = peopleDataSourceHelper
-    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        homeView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(homeView)
+        homeView.peopleDataSource = peopleDataSourceHelper
+
+        homeView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+        homeView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+        homeView.topAnchor.constraint(equalTo: view.topAnchor).isActive = true
+        homeView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        
         navigationItem.title = L10n.sayTheirNames.localizedUppercase
```

Comment:
@hybridcattt Can we have this part in a grouped function,  Otherwise looks good to me 😃